deps/aslam_cameras_april/test/logging/logTest.cpp

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
I included it because it's the Kalibr test for a dependency of the AprilTags library. It looks like the `TagDetector` or one of those similar files calls the Logger, and this is the test for that.

I believe the test was run? I ran it on my computer anyways, maybe I set it up wrong